code/game/machinery/turnstile.dm

```diff
@@ -0,0 +1,66 @@
+/obj/machinery/turnstile
+	name = "turnstile"
+	desc = "A remote control switch."
+	icon = 'icons/obj/turnstile.dmi'
+	icon_state = "turnstile_map"
+	power_channel = ENVIRON
+	density = TRUE
+	armor = list(melee = 50, bullet = 50, laser = 50, energy = 50, bomb = 10, bio = 100, rad = 100, fire = 90, acid = 70)
+	anchored = TRUE
+	use_power = IDLE_POWER_USE

```

Comment:
It's powered, but power doesn't matter for any of its interactions. Make it so that you can force it somehow with a crowbar?

Reply:
It does for allowing you to pass, its default setting is locked unless it scans an okay ID 🤔